Readme.md

```diff
@@ -136,6 +136,22 @@ uvicorn main:app --host 0.0.0.0 --port 8000
 
 Now, your LangChain models and pipelines are accessible via the LangCorn API server.
 
+With a router : 
+```python
+app = FastAPI()
+router = APIRouter()
+
+
+@app.get("/")
+def read_main():
+    return {"message": "Hello World from main app"}
+    
+router: APIRouter = create_service(router,
+    "api.ex1:chain",
+)
+app.include_router(router)
+```
+
```

Comment:
I have not tested it yet but it should be possible to use [mount](https://fastapi.tiangolo.com/advanced/sub-applications/#mounting-a-fastapi-application) 


```python
app = FastAPI()
router = APIRouter()


@app.get("/")
def read_main():
    return {"message": "Hello World from main app"}


lang_app: FastAPI = create_service(
    "api.ex1:chain",
)

app.mount("/subapi", lang_app)

```

Reply:
Hello. 
Yes you are right on this. 

But in my use case cannot define FastAPI application.

Reply:
Would your use case work with `lang_app.router`?

Reply:
Did it solve your issue?

Reply:
I am not able to have access to app for my use case. 
So the only solution is to do my proposal.